EventsExpress/ClientApp/src/components/landing/landing.js

```diff
@@ -52,7 +52,7 @@ class Landing extends Component {
     return (
       <>
         <div className="main">
-          <HeadArticle id={id} onLogoutClick={onLogoutClick} />
+          <HeadArticle id={id} onLogoutClick={onLogoutClick} onSubmit={this.props.onSubmit()}/>
```

Comment:
Are you sure this works? You need to pass in function, not invoke it.

Reply:
Yep, I guess this should be changed

Reply:
Fixed by removing invoking parentheses